delivery/routes.py

```diff
@@ -34,3 +37,27 @@ def put(self, restaurant_id: int):
         restaurant.id = restaurant_id
         updated_restaurant = self.restaurants.update(restaurant)
         return jsonify(self.schema.dump(updated_restaurant)), 200
+
+
+class LoginEndpoint(MethodView):
+    def __init__(self, users, schema):
+        self.users = users
+        self.schema = schema
```

Comment:
by the way, I guess `schema` can be a part of object, not passed from outside. i.e.
```python
def __init__(self, users):
    self.users = users
    self. schema = UserSchema()
```

configurability is nice, but unless it is not too much :)

Reply:
yet, it is also a way to go, a nice pattern when schema is an argument :)

Reply:
I think that better if schema will an argument like in others `MethodView`s